src/plugins/data_view_management/public/components/edit_index_pattern/index_header/index_header.tsx

```diff
@@ -24,50 +25,69 @@ const setDefaultAriaLabel = i18n.translate('indexPatternManagement.editDataView.
 });
 
 const setDefaultTooltip = i18n.translate('indexPatternManagement.editDataView.setDefaultTooltip', {
-  defaultMessage: 'Set as default data view.',
+  defaultMessage: 'Set as default',
+});
+
+const editAriaLabel = i18n.translate('indexPatternManagement.editDataView.editAria', {
+  defaultMessage: 'Edit data view.',
+});
+
+const editTooltip = i18n.translate('indexPatternManagement.editDataView.editTooltip', {
+  defaultMessage: 'Edit',
 });
 
 const removeAriaLabel = i18n.translate('indexPatternManagement.editDataView.removeAria', {
   defaultMessage: 'Delete data view.',
 });
 
 const removeTooltip = i18n.translate('indexPatternManagement.editDataView.removeTooltip', {
-  defaultMessage: 'Delete data view.',
+  defaultMessage: 'Delete',
 });
 
 export const IndexHeader: React.FC<IndexHeaderProps> = ({
   defaultIndex,
   indexPattern,
   setDefault,
+  editIndexPatternClick,
   deleteIndexPatternClick,
   children,
   canSave,
 }) => {
   return (
     <EuiPageHeader
-      pageTitle={<span data-test-subj="indexPatternTitle">{indexPattern.title}</span>}
+      pageTitle={<span data-test-subj="indexPatternTitle">{indexPattern.getName()}</span>}
       rightSideItems={[
+        canSave && (
+          <EuiButton
+            fill
+            onClick={editIndexPatternClick}
+            iconType="pencil"
+            aria-label={editAriaLabel}
+            data-test-subj="editIndexPatternButton"
+          >
+            {editTooltip}
+          </EuiButton>
+        ),
         defaultIndex !== indexPattern.id && setDefault && canSave && (
-          <EuiToolTip content={setDefaultTooltip}>
-            <EuiButtonIcon
-              color="text"
-              onClick={setDefault}
-              iconType="starFilled"
-              aria-label={setDefaultAriaLabel}
-              data-test-subj="setDefaultIndexPatternButton"
-            />
-          </EuiToolTip>
+          <EuiButton
+            onClick={setDefault}
+            iconType="starFilled"
+            aria-label={setDefaultAriaLabel}
+            data-test-subj="setDefaultIndexPatternButton"
+          >
+            {setDefaultTooltip}
+          </EuiButton>
         ),
         canSave && (
-          <EuiToolTip content={removeTooltip}>
-            <EuiButtonIcon
-              color="danger"
-              onClick={deleteIndexPatternClick}
-              iconType="trash"
-              aria-label={removeAriaLabel}
-              data-test-subj="deleteIndexPatternButton"
-            />
-          </EuiToolTip>
+          <EuiButtonEmpty
+            color="danger"
+            onClick={deleteIndexPatternClick}
+            iconType="trash"
+            aria-label={removeAriaLabel}
+            data-test-subj="deleteIndexPatternButton"
```

Comment:
![Screen Shot 2022-04-21 at 12 47 26](https://user-images.githubusercontent.com/7784120/164442719-bba6e857-99ad-4826-8d3a-c562c3a957cf.png)


Should delete dialog also use name?

Apologies if this was discussed before.